include/morphio/types.h

```diff
@@ -45,6 +46,11 @@ namespace Property {
 struct Properties;
 }
 
+namespace vasculature {
+    class VasculatureSection;
+    class VasculatureMorphology;
```

Comment:
This class should be Vasculature
I don't remember what did we say about VasculatureSection, should it be Section (is it possible) ?